.github/workflows/ci.yaml

```diff
@@ -77,6 +77,8 @@ jobs:
       artifact-prefix: packed-charm-cache-true
       cloud: lxd
       juju-snap-channel: 3.3/stable
+      # Workaround for https://github.com/canonical/opensearch-operator/issues/227
+      lxd-snap-channel: 5.20/stable
```

Comment:
@carlcsaposs-canonical this is unnecessary. Check my comment on #227: (1) I fixed the last `limits.network.priority` command and (2) we need to figure out another way to implement this network cut.

Reply:
@carlcsaposs-canonical @Mehdi-Bendriss we don't need to pin, check here: https://github.com/canonical/opensearch-operator/pull/239

And this is a permanent fix moving forward with 5.21+